src/app/location/location.component.scss

```diff
@@ -0,0 +1,39 @@
+.countries-list {
+  max-height: 200px;
+  overflow-y: auto;
+  margin-bottom: 10px;
+  border: 1px solid gray;
+}
+
+.map {
+  width: 100%;
+  min-height: 40vh;
+  height: 0;
+
+  agm-map {
+    height: 100%;
+    width: 100%;
+  }
+}
+
+@media (min-width: 768px) {
+  .buttons-input {
+    width: 200px;
+  }
+
+  .countries-list {
+    max-height: calc(100vh - 120px);
+    overflow-y: auto;
+  }
+
+  .side-data {
+    height: calc(100vh - 58px);
```

Comment:
This looks odd, please explain.

Reply:
That is for adding scroller for part of the page where map, chart and weather data are.
I don't want scroller for whole page, 'couse I have one for country list too, and I need them to be in sync

Reply:
![scroller](https://user-images.githubusercontent.com/61235775/80957994-8c1dd480-8e04-11ea-8575-1a9153eb1af2.png)

Reply:
To avoid this...

Reply:
Could this be done with flex-grow?

Reply:
If it can, I'm not making it work :(